### DISCUSSION TITLE
Downtime for search.serginho.dev

### DISCUSSION THREAD

**Discussion by SergioGlorias** [searxng/searx-instances#278] [Report instance downtime]
The instance was shut down for machine maintenance I will be back as soon as possible.

**Comment by SergioGlorias** [comment]
instance back
but the tor url is still missing (which will be a new one)

**Comment by SergioGlorias** [comment]
fix bugs and new tor URL http://qyyb5xsar7gp4q3agxnrxgajx4nnnihptqhmyee5nolidtkhxjks6ayd.onion

**Comment by unixfox** [comment]
Thank you for your notice!

If you want to update or add the URL of an instance please open an issue.

**Comment by SergioGlorias** [comment]
I know the instance revert to unstable
I'm trying to resolve the issue as quickly as possible.

**Comment by SergioGlorias** [comment]
instance back online

what happened: after maintenance I started using podman instead of docker (although I don't use docker either)
and after having made some changes SearXNG stopped responding
port 8080 was there but not responding
both by curl on localhost and both by caddy

after several attempts I managed to get it to work
